pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -487,41 +493,89 @@ public void callingWithRefreshDelayWithNegativeValueMustError() {
   }
 
   @Test
-  public void callingWithNodesWhitelistOptionButNoValueMustNotError() {
-    parseCommand("--nodes-whitelist");
+  public void callingWithInvalidNodesWhitelistMustDisplayErrorAndUsage() {
+    parseCommand("--nodes-whitelist", "invalid_enode_url");
+    assertThat(commandOutput.toString()).isEmpty();
+    /*
+     Because of the way Picocli handles errors parsing errors for lists with arity 0..*, we don't
+     get the nice error msg with that was wrong. It only shows to the user the values that weren't
+     parsed correctly.
+    */
+    final String expectedErrorOutputStart = "Unmatched argument: invalid_enode_url";
+    assertThat(commandErrorOutput.toString()).startsWith(expectedErrorOutputStart);
+  }
+
+  @Test
+  public void nodesWhitelistOptionMustBeUsed() {
+    parseCommand(
+        "--nodes-whitelist",
+        String.join(",", validENodeStrings),
+        "--bootnodes",
+        validENodeStrings[0]);
 
     verify(mockRunnerBuilder)
         .permissioningConfiguration(permissioningConfigurationArgumentCaptor.capture());
     verify(mockRunnerBuilder).build();
 
-    assertThat(permissioningConfigurationArgumentCaptor.getValue().getNodeWhitelist()).isEmpty();
+    assertThat(
```

Comment:
I would create a List<String> enodeURLAsStringList (or something similar) variable and assign the stream-mapping -collect result to it.
This will make the assertion cleaner:
`assertThat(enodeURLAsStringList).containsExactlyInAnyOrder(validENodeStrings);`